### DISCUSSION TITLE
Support for ControlNet T2I CoAdapter/Fuser

### DISCUSSION THREAD

**Discussion by setothegreat** [vladmandic/sdnext#656] [Ideas]
Not sure if it would be more appropriate to ask here or in the ControlNet WebUI Extension discussion page, since I don't know how much this implementation of ControlNet differs nor how much could be adapted here, but thought I would ask anyways.

T2I has a (relatively) new adapter that they've released with their models called "Fuser". It seems to essentially act like current WebUI implementations of MultiControlNet, only it has awareness of the other models being concurrently run and thus seems to produce better results.

I'm not a programmer, and have no idea how one would go about implementing such a model into the current infrastructure, but the [GitHub page does include the code and downloadable local demos that could help with this](https://github.com/TencentARC/T2I-Adapter).

Seems like it could be valuable to implement (if it's even possible to do so).

**Comment by vladmandic** [comment]
CN here is a link to original, not a separate copy.
CN is developing fast and I don't want to have local modifications thar would slow down of adoption of main CN.

As such, best to create this request with original CN repo.
